src/test/java/seedu/cakecollate/logic/commands/CommandTestUtil.java

```diff
@@ -100,6 +111,14 @@
                 .withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB).withAddress(VALID_ADDRESS_BOB)
                 .withOrderDescriptions(VALID_BERRY_ORDER).withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND)
                 .withDeliveryDate(VALID_DELIVERY_DATE_BOB).build();
+
+
+        try {
+            temp = ParserUtil.parseIndexList(INDEXES_2);
+        } catch (ParseException e) {
+            temp = null;
+        }
```

Comment:
Is there a reason to create an index list through ParseUtil.parseIndexList? Since other tests uses this static value ORDER_ITEM_INDEXLIST2 I think making it such that theres lower chance it will break will be better i.e. create the two indexes and then create the indexlist instead of depending on ParseUtil to create it.

Reply:
Ah you're right. Did this to save some lines of clutter but yeah sounds best to avoid depending on other production class 😅